cirq/protocols/json_test_data/LinearDict.repr

```diff
@@ -0,0 +1 @@
+[cirq.LinearDict({}), cirq.LinearDict({'a': 1}), cirq.LinearDict({'a': 1, 'b': 2}), cirq.LinearDict({cirq.X: 1.0, cirq.Y: 2})]
```

Comment:
Why doesn't this cause the repr roundtrip test to fail? As I understand it this is the repr for a list of LinearDict rather than a LinearDict

Reply:
The test allows you to write a list of reprs.  See line 488 in json_serialization_test.py

Reply:
TIL. Thanks!